Core/StatisticsLoader.swift

```diff
@@ -22,26 +22,78 @@ import Foundation
 
 public class StatisticsLoader {
     
+    public typealias Completion =  (() -> ())
```

Comment:
This feels redundant. Personally, I would avoid type aliases for functions unless they are more complex than this simple case or if they are going to be stored as a variable.

Either way, should return `Void` rather than `()`

Reply:
I've switched to Void. I'm with you on the Completion being a potential overkill. I pulled it out as the code was getting a bit cumbersome with the escaping prefix and the default return parameter but would normally leave it.